src/main/java/Duke.java

```diff
@@ -1,10 +1,68 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        System.out.println("____________________________________________________________\n" +
+                "Hello! I'm Duke\n" +
+                "What can I do for you?\n"
+        );
+        ArrayList<Task> items = new ArrayList<Task>();
+        String input = "";
+
+        while (!input.equals("bye")) {
+            input = sc.next();
+            int order;
+            Task newTask;
+            String prompt;
+            String description;
+
+            if (input.equals("mark")) {
+                order = sc.nextInt();
+                items.get(order - 1).setMark(true);
+                System.out.println("____________________________________________________________\n" + "  " + "Nice! I've marked this task as done:\n" + "[X] " + items.get(order - 1).getDescription() + "\n" + "____________________________________________________________\n");
+
+            } else if (input.equals("unmark")) {
+                order = sc.nextInt();
+                items.get(order - 1).setMark(false);
+                System.out.println("____________________________________________________________\n" + "  " + " OK, I've marked this task as not done yet:\n" + "[X] " + items.get(order - 1).getDescription() + "\n" + "____________________________________________________________\n");
+            } else if (input.equals("list")) {
+                System.out.println("\t____________________________________________________________\r\n"
+                            + "\t Here are the tasks in your list:");
+                    for (int i = 0; i < items.size(); i++) {
+                        System.out.println("\t " + (i + 1) + ". " + items.get(i).toString());
+                    }
+                    System.out.println("\n\t____________________________________________________________\r\n");
+            } else if (input.equals("todo")){
```

Comment:
```suggestion
            } else if (input.equals("todo")) {
```
Try to keep the spacing constant throughout your code for easier readability